ivi-layermanagement-examples/multi-touch-viewer/src/multi-touch-viewer.c

```diff
@@ -641,39 +643,65 @@ touch_event_test_main(struct touch_event_test_params *p_params)
     return 0;
 }
 
-void
+static void
 usage(int status)
 {
-    printf("usage: multi-touch-viewer [OPTION]\n");
-    printf("  -p : print received touch point\n");
+    fprintf(stderr, "    -h,  --help              display this help and exit.\n"
+                    "    -p,  --print-log         print received touch point.\n"
+                    "    -s,  --surface-type      select the type of surface, default is the ivi_app_surface:\n"
```

Comment:
Instead of surface type, shell type is more known in the community i.e. wl_shell, xdg_shell, ivi_shell. It is more readable in my opinion to provide shell type.

Reply:
Yes. I have just update from surface type to shell type.